openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -0,0 +1,169 @@
+import unittest
+import ddt
+
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.conf import settings
+from rest_framework.test import APITestCase, APIClient
+
+from student.tests.factories import UserFactory
+from student.models import UserProfile
+
+TEST_PASSWORD = "test"
+
+@ddt.ddt
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
```

Comment:
We can add it to the CMS urls.py when it is actually used in Studio. For now, I didn't see a reason to.